### DISCUSSION TITLE
Multi Vendor

### DISCUSSION THREAD

**Discussion by mwamodo** [getcandy/candy-api#372] [Help] [answered]
Hello,

I want to do a multi vendor or a multi store eCommerce project with the API (a single store front with multiple "vendors" or shops adding products). What is the best way to approach the challenge?

**Comment by glennjacobs** [comment, marked as answer]
Would they share the same products or have their own? If they share products, then Channels would be the suggested solution. If you need a multi-tenancy set-up, then I would suggest looking at https://tenancyforlaravel.com/

**Reply by mwamodo** [reply]
They would have their own products, with an admin that would be able to manage all the products. Thank you I will take a look at tenancy for laravel.

**Reply by Tofandel** [reply]
There is also [Tenancy/Tenancy](https://tenancy.dev/) my personnal favourite
